sync_diff_inspector/config.go

```diff
@@ -29,17 +30,53 @@ const (
 	percent100 = 100
 )
 
-// TableCheckCfg is the config of table to be checked.
-type TableCheckCfg struct {
-	// table name
-	Name string `toml:"name"`
-	// Schema is seted in SourceDBCfg
-	Schema string
+// DBConfig is the config of database, and keep the connection.
+type DBConfig struct {
+	dbutil.DBConfig
+
+	Label string `toml:"label" json:"label"`
```

Comment:
Can you explain what's Label and what's its usage?

Reply:
in sharding mode, have many datasource, use label identify a source

Reply:
Unify name, how about instance id